File: src/main/java/com/tobe/healthy/lessonhistory/domain/dto/in/UnwrittenLessonHistorySearchCond.java
package com.tobe.healthy.lessonhistory.domain.dto.in;

import com.tobe.healthy.lessonhistory.domain.entity.WritingStatus;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UnwrittenLessonHistorySearchCond {

    private String lessonDate;
    private Long studentId;
    private WritingStatus writingStatus;
}

File: src/main/kotlin/com/tobe/healthy/lessonhistory/domain/dto/in/UnwrittenLessonHistorySearchCond.kt
package com.tobe.healthy.lessonhistory.domain.dto.`in`

import com.tobe.healthy.lessonhistory.domain.entity.WritingStatus

data class UnwrittenLessonHistorySearchCond(
    val lessonDate: String? = null,
    val studentId: Long? = null,
    val writingStatus: WritingStatus? = null
)